iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveTableUtil.java

```diff
@@ -225,4 +231,34 @@ public static boolean isCtas(Properties properties) {
     return Boolean.parseBoolean(properties.getProperty(hive_metastoreConstants.TABLE_IS_CTAS));
   }
 
+  protected static Properties getProps() {
+    Properties props = new Properties();
+    props.put(serdeConstants.SERIALIZATION_FORMAT, "" + Utilities.tabCode);
+    props.put(serdeConstants.SERIALIZATION_NULL_FORMAT, "NULL");
+    return props;
+  }
+
+  protected static String getParseData(String parseData, String specId) throws JsonProcessingException {
+    ObjectMapper mapper = new ObjectMapper();
+    Map<String, String> map = mapper.readValue(parseData, Map.class);
+    String partString = "";
+    for (Map.Entry<String, String> entry : map.entrySet()) {
+      if (entry.getValue() != null) {
+        String partition = String.format("%s=%s", String.valueOf(entry.getKey()), String.valueOf(entry.getValue()));
+        partString = partString.length() > 0 ? partString + "/" + partition : partition;
+      }
+    }
+    return String.format("Spec-id=%s/%s", specId, partString);
+  }
+
+  protected static JobConf getJobConf(Configuration confs, Path path, org.apache.hadoop.hive.ql.metadata.Table tbl) {
+    JobConf job = new JobConf(confs);
+    job.set("mapred.input.dir", path.toString());
```

Comment:
could we use protective HiveConf.setxx() instead of job.set?

Reply:
Done.